packages/gradle-plugin/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/GenerateCodegenSchemaTaskTest.kt

```diff
@@ -65,14 +66,10 @@ class GenerateCodegenSchemaTaskTest {
 
     val task = createTestTask<GenerateCodegenSchemaTask> { it.jsRootDir.set(jsRootDir) }
 
-    assertEquals(jsRootDir, task.jsInputFiles.dir)
-    assertEquals(
-        setOf(
-            "**/build/**/*",
-        ),
-        task.jsInputFiles.excludes)
-    assertEquals(1, task.jsInputFiles.files.size)
-    assertEquals(setOf(File(jsRootDir, "afolder/includedfile.js")), task.jsInputFiles.files)
+    assertThat(task.jsInputFiles.dir).isEqualTo(jsRootDir)
+    assertThat(task.jsInputFiles.excludes).isEqualTo(setOf("**/build/**/*"))
+    assertThat(task.jsInputFiles.files).hasSize(1)
```

Comment:
This is unnecessary now 👍

Reply:
should I delete those assertions? Which means you don't need assertion for `task.jsInputFiles`?

Reply:
I meant only the size assertion